src/api/firebase.js

```diff
@@ -65,12 +72,13 @@ export async function addItem(listId, { itemName, daysUntilNextPurchase }) {
 	});
 }
 
-export async function updateItem() {
-	/**
-	 * TODO: Fill this out so that it uses the correct Firestore function
-	 * to update an existing item. You'll need to figure out what arguments
-	 * this function must accept!
-	 */
+export async function updateItem(listId, itemId, currentItemUpdates) {
```

Comment:
Great job writing the `updateItem` function! I wouldn't change anything. Easy to read :)